webpack.config.js

```diff
@@ -20,6 +20,7 @@ var wpconfig = {
     entry: {
         CourseOutline: './openedx/features/course_experience/static/course_experience/js/CourseOutline.js',
         CourseSock: './openedx/features/course_experience/static/course_experience/js/CourseSock.js',
+        CoursetalkReviews: './openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js',
```

Comment:
FYI: If you rename "CoursetalkReviews.js" to "CourseTalkReviews.js", let that live on a separate commit that doesn't get squashed to ensure history remains accurate.